packages/react-components/react-storybook/src/decorators/withFluentVrTestVariants.tsx

```diff
@@ -0,0 +1,55 @@
+import * as React from 'react';
+import { FluentProvider } from '@fluentui/react-provider';
+import { teamsHighContrastTheme, webDarkTheme, webLightTheme } from '@fluentui/react-theme';
+import { makeDecorator } from '@storybook/addons';
+import type { StoryContext } from '@storybook/addons';
+
+export const DARK_MODE = 'DarkMode';
+export const HIGH_CONTRAST = 'HighContrast';
+export const RTL = 'Rtl';
+
+// MakeDecoratorOptions is not an exported interface by storybook so had to include necessary options
+interface AssertedMakeDecoratorOptions {
+  name: string;
+  parameterName: string;
+  skipIfNoParametersOrOptions?: boolean;
+  wrapper: (
+    storyFn: (context: StoryContext) => React.ReactNode,
+    context: StoryContext,
+    settings: {
+      parameters: typeof DARK_MODE | typeof HIGH_CONTRAST | typeof RTL;
+    },
+  ) => React.ReactNode;
+}
+
+type AssertedMakeDecorator = ({
+  name,
+  parameterName,
+  skipIfNoParametersOrOptions,
+  wrapper,
+}: AssertedMakeDecoratorOptions) => () => React.ReactNode;
+
+export const withFluentVrTestVariants = ((makeDecorator as unknown) as AssertedMakeDecorator)({
```

Comment:
is the `makeDecorator` function officially supported ? according to latest docs we should be returning a pure function and using hooks like `useGlobal` and `useParameters` to access context

https://storybook.js.org/tutorials/create-an-addon/react/en/decorators/

Reply:
Yup - will update the implementation